.github/workflows/check-pkg-availability.yml

```diff
@@ -0,0 +1,129 @@
+              exit 1
+              ;;
+          esac
+
+          # Set repo based on distrib
+          echo "Checking if stable packages are available for $COMBINATION"
+          case "$COMBINATION" in
+            almalinux:8:23.10 | almalinux:8:23.04 | almalinux:8:22.10 | almalinux:8:22.04)
+              dnf install -y epel-release
+              dnf install -y dnf-plugins-core
+              dnf config-manager --set-enabled powertools
+              dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+              dnf config-manager --add-repo https://packages.centreon.com/rpm-standard/$MAJOR_VERSION/el8/centreon-$MAJOR_VERSION.repo
+              dnf config-manager --set-enabled centreon-$MAJOR_VERSION-stable
+              dnf install -y https://rpms.remirepo.net/enterprise/remi-release-8.rpm
+              dnf module reset -y php
+              dnf module install -y php:${PHP_REPO}
+              dnf install -y curl sudo
+              curl -LsS https://r.mariadb.com/downloads/mariadb_repo_setup \
+                | bash -s -- --os-type=rhel --skip-check-installed --os-version=8 --mariadb-server-version="mariadb-10.5"
+              dnf install -y mariadb-server mariadb
+
+              if [[ "${{ inputs.module }}" == "web" ]];then
+                dnf download --resolve -y centreon-${{ inputs.module }}
+              else
+                dnf download --resolve -y centreon
+                dnf download --resolve -y centreon-${{ inputs.module }}
+              fi
+            ;;
+            almalinux:9:23.10 | almalinux:9:23.04)
+              dnf install -y dnf-plugins-core
+              dnf install -y epel-release
+              dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm
+              dnf config-manager --set-enabled crb
+              dnf config-manager --add-repo https://packages.centreon.com/rpm-standard/$MAJOR_VERSION/el9/centreon-$MAJOR_VERSION.repo
+              dnf config-manager --set-enabled centreon-$MAJOR_VERSION-stable*
+              dnf module reset -y php
+              dnf module install -y php:$(cut -d '-' -f 2 <<< ${PHP_REPO})
+              dnf install -y --allowerasing sudo curl
+              curl -LsS https://r.mariadb.com/downloads/mariadb_repo_setup \
+                | sudo bash -s -- --os-type=rhel --os-version=9 --mariadb-server-version="mariadb-10.5"
+              dnf install -y mariadb-server mariadb
+
+              if [[ "${{ inputs.module }}" == "web" ]];then
+                dnf download --resolve -y centreon-${{ inputs.module }}
+              else
+                dnf download --resolve -y centreon
+                dnf download --resolve -y centreon-${{ inputs.module }}
+              fi
+            ;;
+            debian:bullseye:23.10 | debian:bullseye:23.04 | debian:bullseye:22.10 | debian:bullseye:22.04)
+              apt update && apt upgrade -y
```

Comment:
`apt upgrade` is useful ?
The workflow will be much more slower with this

Reply:
indeed, I m removing it